numba_dpex/tests/core/types/DpctlSyclQueue/test_box.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2020 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Tests for boxing for dpnp.ndarray
+"""
+
+import dpnp
+import pytest
+from dpctl import SyclQueue
+
+from numba_dpex import dpjit
+
+
+@pytest.mark.xfail(reason="dpjit allocates new dpctl.SyclQueue on boxing")
+def test_boxing_without_parent():
+    """Tests basic boxing and unboxing of a dpnp.ndarray object.
+
+    Checks if we can pass in and return a dpctl.ndarray object to and
+    from a dpjit decorated function.
+    """
+
+    @dpjit
+    def func(arr):
+        queue = arr.sycl_queue
+        return queue
+
+    arr = dpnp.empty(10)
+
+    q = func(arr)
+
+    assert q is not None
+    assert isinstance(q, SyclQueue)
+    assert id(q) == id(arr.sycl_queue)  # this must work after dpjit fix
```

Comment:
Only the last assert is xfail right? The other asserts should pass?

Reply:
Yes, is there a way to achieve it?

Reply:
You will have to write two separate test cases.

Reply:
I've been thinking about what test should actually check and updated tests a little bit. So this one is always pass, and another one is failing. Thx